themes/elos/views/shared/_recording_row.html.erb

```diff
@@ -13,16 +13,16 @@
       </div>
     </div>
   </td>
-  <td class="col-2 align-middle">
+  <td class="col-6 col-md-2 align-middle td-meeting-date">
```

Comment:
I removed 1 or 2 classes such as `td-meeting-date` because they were not used anywhere.